runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineOptions.java

```diff
@@ -34,6 +34,12 @@
  */
 public interface SparkPipelineOptions extends SparkCommonPipelineOptions {
 
+  @Description("The directory to save Spark History Server logs")
+  @Default.String("/tmp/spark-events/")
```

Comment:
Perhaps we should also add an option like `isEventLogEnabled` and make it `false` by default, like Spark's `spark.eventLog.enabled` property.

Reply:
Ok, I've done it.